crypto/kzg/Cargo.toml

```diff
@@ -16,8 +16,8 @@ serde_derive = "1.0.116"
 ethereum_serde_utils = "0.5.0"
 hex = "0.4.2"
 ethereum_hashing = "1.0.0-beta.2"
-c-kzg = { git = "https://github.com/ethereum/c-kzg-4844", rev = "13cec820c08f45318f82ed4e0da0300042758b92" , features = ["mainnet-spec"]}
-c_kzg_min = { package = "c-kzg", git = "https://github.com/ethereum//c-kzg-4844", rev = "13cec820c08f45318f82ed4e0da0300042758b92", features = ["minimal-spec"], optional = true }
+c-kzg = { git = "https://github.com/ethDreamer/c-kzg-4844", rev = "483d460c164967c36f7377c87be0fb60fad2ff99" , features = ["mainnet-spec"]}
```

Comment:
This version only makes some [minor necessary adjustments](https://github.com/ethDreamer/c-kzg-4844/commit/483d460c164967c36f7377c87be0fb60fad2ff99) to the existing library. It should be trivial to get these changes included upstream.

I think we also want to eliminate stack allocations though & that would [require an additional change](https://github.com/ethDreamer/c-kzg-4844/commit/9ac5019506c7389133a6019e78773349a268ef85) which might be more challenging since it involves `unsafe` code.

Reply:
@pawanjay176 says his PR segfaults of devnets when he adds passing my reference to these methods